docs/product-user-guide/src/main/asciidoc/business-central-settings-changing-permissions-proc.adoc

```diff
@@ -0,0 +1,18 @@
+[id='business-central-settings-changing-permissions-proc']
+= Changing permissions
```

Comment:
Given the limitation that you mention in the opening section, I'd change the title to "Changing permissions for groups and roles"

Reply:
Done